shell/platform/darwin/macos/framework/Source/FlutterEmbedderExternalTextureTest.mm

```diff
@@ -60,6 +60,7 @@ - (CVPixelBufferRef)pixelBuffer {
   CVReturn status = CVPixelBufferCreate(kCFAllocatorDefault, _width, _width, _pixelFormatType,
                                         (__bridge CFDictionaryRef)options, &pxbuffer);
   NSAssert(status == kCVReturnSuccess && pxbuffer != NULL, @"Failed to create pixel buffer.");
+  FML_DCHECK(status == kCVReturnSuccess && pxbuffer != nullptr) << "Failed to create pixel buffer";
```

Comment:
Any reason not to switch this just to a `FML_DCHECK`?

Reply:
Could you remove the `NSAssert` then? It sounds like we would ideally migrate them all to `FML_*CHECK`.